src/main/java/cs/ualberta/cmput402/tictactoe/scoreboard/Scoreboard.java

```diff
@@ -0,0 +1,36 @@
+package cs.ualberta.cmput402.tictactoe.scoreboard;
+
+public class Scoreboard {
```

Comment:
This was a great idea to make a scoreboard class to keep track of the running scores. Makes the code more modular and easier to write unit tests for the logic to validate different scenarios.